src/lib/features/onboarding/onboarding-read-model.ts

```diff
@@ -4,14 +4,19 @@ import type {
     InstanceOnboarding,
     ProjectOnboarding,
 } from './onboarding-read-model-type';
-import { millisecondsToMinutes } from 'date-fns';
 
-const calculateTimeDifferenceInMinutes = (date1?: Date, date2?: Date) => {
-    if (date1 && date2) {
-        const diffInMilliseconds = date2.getTime() - date1.getTime();
-        return millisecondsToMinutes(diffInMilliseconds);
-    }
-    return null;
+const instanceEventLookup = {
```

Comment:
Need mapping for new column format

Reply:
maybe we can use the - notation for the metrics so that we can skip mapping?

Reply:
did we add camel case to be consistent with other metrics?

Reply:
I just dont want to mess with json object with - in the key.